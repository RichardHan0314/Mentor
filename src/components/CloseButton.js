import React from 'react';
import styled from 'styled-components';

const CloseButton = () => {
    return (
        <CloseBtn/>
    );
};

const CloseBtn = styled.div`
    width: 30px;
    height: 30px;
    position: absolute;
    top: 20px;
    right: 20px;

    &::after {
        content: '';
        height: 30px;
        border-left: 2px solid #fff;
        position: absolute;
        transform: rotate(45deg);
        left: 14px;
    }

    &::before {
        content: '';
        height: 30px;
        border-left: 2px solid #fff;
        position: absolute;
        transform: rotate(-45deg);
        left: 14px;
    }
`;

export default CloseButton;
