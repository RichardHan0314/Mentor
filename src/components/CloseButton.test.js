import React from 'react';
import { render } from '@testing-library/react';
import CloseButton from './CloseButton';

describe('CloseButton component', () => {
    test('Render CloseButton snapshot', () => {
        const { asFragment } = render(<CloseButton />);
        expect(asFragment(<CloseButton />)).toMatchSnapshot();
    });
});
