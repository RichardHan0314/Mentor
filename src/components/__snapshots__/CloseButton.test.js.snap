// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackToHomeBtn component Render BackToHomeBtn snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm dmMcfw"
  />
</DocumentFragment>
`;

exports[`BackToHomeBtn component render BackToHomeBtn snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm dmMcfw"
  />
</DocumentFragment>
`;

exports[`CloseButton component Render CloseButton snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm dmMcfw"
  />
</DocumentFragment>
`;
